grakn-core/src/main/java/ai/grakn/graql/analytics/ClusterQuery.java

```diff
@@ -43,6 +45,10 @@
      */
     ClusterQuery<Map<String, Set<String>>> members();
 
+    boolean membersSet();
```

Comment:
`memberSet` returns a boolean?? Java docs explaining what this means.

Reply:
I think it indicates whether the query return members of each cluster or just the size of the cluster. Though I think maybe we should delete the method for returning size at some point? what do you guys think?

Reply:
Renamed to `isMembersSet` and added Javadoc ✅

Reply:
@duckofyork does that mean it would only ever return cluster members? Yeah, I agree. Just size seems weird.

Reply:
The idea was when the graph is huge, the user would have to use java api, so by default graql console returns size of each cluster. I will submit another pr after this one got merged.